tests/scripts/download_gherkin_features.bat

```diff
@@ -0,0 +1,6 @@
+mkdir .\tests\tempFeatures
+curl https://codeload.github.com/elastic/apm/zip/master -o .\tests\tempFeatures\features.zip
```

Comment:
is `codeload.github.com` the official and supported URL?

Reply:
It's the URL they use when you click on the download button on the website. We use it for a long time already and never had problems (other than the occasional network glitch)

Reply:
I was not familiar with that particular entrypoint, then I searched and found that's not officially supported but some internal entrypoint which  it could potentially be changed. The reference I found is below one:
- https://github.com/adobe/git-server/issues/5#issuecomment-403072428

We could potentially use `curl -L https://github.com/elastic/apm/archive/master.zip` to fetch the artifact, similar to what we do in the apm-integration-testing, what do you think?

Reply:
Sure, although I don't anticipate them changing those URLs anytime soon, they are mentioned in the docs, even if only as en example response: https://developer.github.com/v3/repos/contents/#get-archive-link

Reply:
It's not clear what they do support or not! Anyway, no strong opinion, up to @ElWPenn  how to proceed then. I'm glad to know a bit more about this! Thanks Beni for the feedback